terraform_cft/01_shared_services/cloud_function.tf

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2019 Google LLC. This software is provided as-is, without warranty
+ * or representation for any use or purpose. Your use of it is subject to your 
+ * agreement with Google.  
+ */
+module "proxy_autoscale_event" {
+  # FIXME: the current version of this module published under the terraform registry does not
+  # support the include_children option. Using a local version of the module until this PR makes
+  # its way to the registry.
+  # Pull request: https://github.com/terraform-google-modules/terraform-google-event-function/pull/39
+  source = "../../../terraform-google-event-function/modules/event-folder-log-entry"
```

Comment:
Instead of referencing a local version, you can reference a git branch.

Reply:
Replaced this local reference to the branch that contains the nre field.